fog/overseer/server/src/service.rs

```diff
@@ -56,7 +54,7 @@ where
             self.recovery_db.clone(),
             self.logger.clone(),
             self.is_enabled.clone(),
-            self.stop_requested.clone(),
+            Arc::new(AtomicBool::new(false)),
```

Comment:
Why pass this as an argument? It's an internal implementation detail of `OverseerWorker`

Reply:
Agreed. Done.